src/System.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -32,6 +40,62 @@ public static bool AreConnStringsSetup()
             return !string.IsNullOrEmpty(NpConnStr) && !string.IsNullOrEmpty(TcpConnStr);
         }
 
+        public static bool IsServiceBrokerEnabled()
+        {
+            if (!serviceBrokerEnabled.HasValue)
+            {
+                serviceBrokerEnabled = false;
+                if (AreConnStringsSetup())
+                {
+                    try
+                    {
+                        var builder = new SqlConnectionStringBuilder(TcpConnStr);
+                        string database = builder.InitialCatalog;
+                        builder.ConnectTimeout = 2;
+                        using (var connection = new SqlConnection(builder.ToString()))
+                        using (var command = new SqlCommand("SELECT is_service_broker_enabled FROM sys.sys.databases WHERE name=@name", connection))
```

Comment:
Here's a bug. The name of the DB is hardcoded to @name. When I try to run the tests on my server where SB is enabled, the tests are skipped. Please fix this to pass in the correct DB name